### DISCUSSION TITLE
Release 2.4.6 is missing a tar.gz file in the assets

### DISCUSSION THREAD

**Discussion by cedricroijakkers** [OpenIDC/mod_auth_openidc#545] [General]
I was trying to build release 2.4.6 locally, but there is no tarball to be downloaded.

Downloading the tar snapshot of the git repository does not work either, as the configure script is missing.

The tarball is present in release 2.4.5 and older releases, can it also be added in 2.4.6?

**Comment by zandbelt** [comment]
I just added it: https://github.com/zmartzone/mod_auth_openidc/releases/download/v2.4.6/mod_auth_openidc-2.4.6.tar.gz
